x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,44 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Coin,  // price to pay for each period, Coin contains both amount and denomination
+  expiration_time int,  // the timestamp in seconds when plan expires, it will be removed automatically
```

Comment:
so this is like "global" expiration time and each subscription has its own expiration time? 
maybe both options are possible as per @christopher-crypto -- in here, maybe it'll make sense to have something like "duration" in Plan and subscription's expiration will be "block_time + plan duration" ?

Reply:
this is the expiration time of the whole plan, the whole plan is stopped after this time.

Reply:
Plan don't expire now.